extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -206,6 +213,12 @@ export class SubscriptionService {
     ));
     entitlement.setCurrentProduct(productId);
     this.platformStore_.resolveEntitlement(serviceId, entitlement);
+    this.subscriptionAnalytics_.event(
+        SubscriptionAnalyticsEvents.ENTITLEMENT_RESOLVED,
+        {
+          'platform-id': serviceId,
```

Comment:
Let's name this var `serviceId`. Since we call it `service: []` in config, that'd read cleaner.

Also, I'd like to make this more standardized. How about `SubAnalytics.event()` were to accept `serviceId` arg?

Reply:
Changed to `serviceId`

Added a new method `serviceEvent` because some event do not have serviceId like `STARTED`